src/index.html

```diff
@@ -1,15 +1,31 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Quiz App</title>
     <link rel="icon" type="image/png" href="assets/favicon.png" />
     <link rel="stylesheet" href="styles/styles.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@100&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <header class="app-header">
       <h1 class="app-header__heading">Quiz App</h1>
     </header>
+    <section class="card">
+      <h2 class="card__question">Frage</h2>
+      <button class="card__button">Zeige mir die Antwort</button>
+      <p class="card__answer">Wie viele Zähne hat ein Eisbär?</p>
+      <ul class="card__tag-list">
+        <li>32</li>
+        <li>42</li>
+        <li>52</li>
+        <li>62</li>
+      </ul>
+    </section>
```

Comment:
The main content of your website should be wrapped in a main element.
```suggestion
   <main>
    <section class="card">
      <h2 class="card__question">Frage</h2>
      <button class="card__button">Zeige mir die Antwort</button>
      <p class="card__answer">Wie viele Zähne hat ein Eisbär?</p>
      <ul class="card__tag-list">
        <li>32</li>
        <li>42</li>
        <li>52</li>
        <li>62</li>
      </ul>
    </section>
  </main>
```